specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-11-15-preview/managedCassandra.json

```diff
@@ -0,0 +1,1855 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CommandPostBody"
+            },
+            "description": "Specification which command to run where"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success. The command operation will begin as soon as possible.",
+            "schema": {
+              "$ref": "#/definitions/CommandPublicResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/commands": {
+      "post": {
+        "operationId": "CassandraClusters_ListCommand",
+        "description": "List all commands currently running on ring info",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraListCommand": {
+            "$ref": "./examples/CosmosDBManagedCassandraListCommand.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of shell commands.",
+            "schema": {
+              "$ref": "#/definitions/ListCommands"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/commands/{commandId}": {
+      "post": {
+        "operationId": "CassandraClusters_GetCommandAsync",
+        "description": "Get list of details about commands that were run asynchronously. Use \"all\" as commandId to list all commands in three days associated with the cluster.",
```

Comment:
The description, response text etc. are all indicating it is for list of commands, whereas the URL is for commandId.